urdfenvs/mobileReacher/resources/mobileRobot.py

```diff
@@ -25,17 +25,17 @@ def setJointIndices(self):
             self.urdf_joints = [0, 1, 2, 4, 5, 6, 7, 8, 9, 10]
 
     def reset(self, pos=None, vel=None):
-        self._vels_int = vel
+        self._integratedVelocities = vel
         return super().reset(pos=pos, vel=vel)
 
-    def setAccLimits(self):
+    def setAccelerationLimits(self):
         accLimit = np.array([1.0, 1.0, 1.0, 15.0, 7.5, 10.0, 12.5, 15.0, 20.0, 20.0, 1.0, 1.0])
         self._limitAcc_j[0, :] = -accLimit[0:self._n]
         self._limitAcc_j[1, :] = accLimit[0:self._n]
 
-    def apply_acc_action(self, accs, dt):
-        self._vels_int += dt * accs
-        self.apply_vel_action(self._vels_int)
+    def apply_acceleration_action(self, accs, dt):
```

Comment:
I disagree, but do want to take naming conventions to a higher level, a consistent naming style. 
for example: camelCase for python files, PascaleCase for classes and snake_case for functions.

Reply:
Makes sense. We should agree on one style and stick to that everywhere.
So `apply_acceleration_action` does not fit in any style book, so that should be changed.
My  suggestion is:
Let's all have a read at the pep guide: https://realpython.com/python-pep8/#tips-and-tricks-to-help-ensure-your-code-follows-pep-8
Then we write a summary for this project in the repository description.
And then, I'll change everything in this PR accordingly.